GLOSSARY.md

```diff
@@ -0,0 +1,31 @@
+# Glossary
+
+This document gathers the canonical nomenclature of the Lightning concepts used by the LDK contributors. Note, they may diverge from the BOLT specs or the ones used by the wider Lightning community, the test of time has revealed that it can be hard to agree on terms employed. The following terms are employed with a best effort across the codebase aiming to reduce both code reviewers and library users confusion.
+
+### Channel Initiator
+
+In LDK context, the channel initiator is the entity who is taking the decision to open a channel. Finalization is relying on counterparty channel acceptance policy. Within current protocol version, the initiator has the burden to pay channel onchain fees.
+
+### Holder/Counterparty
+
+Inspired by financial cryptography, the holder is the entity beneficiary of a right. In LDK context, the holder is the entity who is served by the implementation in its channel operations. The counterparty is the entity participating in the channel operation as a peer, but of whom the interests are not of concern by the implementation.
+
+Used across the channel-management subsystem (`Channel`).
```

Comment:
`Channel` is _part_ of a subsystem which also includes `ChannelManager`, right?

Reply:
Re-worded, lmk if clearer.

Reply:
Thanks, the phrasing is better but I think you can stick with "management" here.